INSTALL.md

```diff
@@ -68,6 +68,25 @@ source, building and installing them. They can be located here:
 
 To get md2man-roff, see there page at: <https://github.com/sunaku/md2man>
 
+#### Fedora
+
+Here we are going to satisfy tpm2-tools dependencies with:
+* tpm2-tss: <https://github.com/tpm2-software/tpm2-tss>
+* tpm2-abrmd: <https://github.com/tpm2-software/tpm2-abrmd>
+* TPM simulator: <https://downloads.sourceforge.net/project/ibmswtpm2/ibmtpm974.tar.gz>
+
+Which are necessary for the build example section at the bottom of this file, we need to satisfy the dependencies for each item named above except for the simulator, in the next code block we install a mix of packages that should get the work done.
+
```

Comment:
Please move all the above either to the ***Building*** section or before the ***Ubuntu 16.04*** section.